### DISCUSSION TITLE
Reordering of multiselect (hasMany) relationship & select fields

### DISCUSSION THREAD

**Discussion by Que-tin** [payloadcms/payload#691] [Feature Requests & Ideas]
Currently, as far as I know there is no way to reorder the relatioship entries in a field with `hasMany: true`. Had this feature being requested a couple times by customers in the past (not with payload) and I think it would be a good addition to the already pretty good relationship field.

Not sure what would be the best way to implement it, maybe making the `hasMany` key accept an object with Type MultiSelectProps or smth. so it can also be reused on normal multi selects outside of the relationship context.

**Comment by jmikrut** [comment]
Hey @Que-tin — this is a great request, and we actually have this one on our radar already. You can expect this coming from the core Payload team at some point in the future but if you wanted to PR, there is an example from the `react-select` docs that shows how to do this:

https://react-select.com/advanced#sortable-multiselect

**Reply by jmikrut** [reply]
Just for anyone else coming here, this is now supported by default and merged into Payload. 👍
